<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindAndReplace</class>
 <widget class="QDialog" name="FindAndReplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Window</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>461</width>
     <height>231</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btnFind">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReplace">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editFind">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="editFindAndReplace">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Find:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFindAll">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Find all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReplaceAll">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Replace all</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkDirectionBackwards">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Use direction backwards </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkCaseSensitive">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Case sensitive</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../example.qrc"/>
 </resources>
 <connections/>
</ui>
